python/tdnfbase.c

```diff
@@ -19,7 +19,7 @@ base_dealloc(PPY_TDNF_BASE self)
 static PyObject *
 base_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 {
-    PPY_TDNF_BASE self = NULL;
+    PPY_TDNF_BASE self;
```

Comment:
please keep the init. it might look redundant as the next line here is assigning value to self. however, code could change later. if left uninitialized, it is more of a burden for review. so please leave init code as is here and elsewhere

Reply:
I partially agree with you on this one, yes initialising variables during declaration is a good approach(in defensive programming) and there's absolutely no harm in it AFAIK. But IMO a programmer should know what he is doing and and he should be in total control. I'll make the modifications as per your suggestions and push the changes, I dis this because that's what linux kernel code has taught me so far, same goes with not using curly braces in single line if else blocks :)